global_helpers/panther_greynoise_helpers.py

```diff
@@ -55,6 +73,13 @@ def url(self, match_field) -> str:
         return f"https://www.greynoise.io/viz/ip/{deep_get(self.noise, match_field, 'ip')}"
 
     def context(self, match_field) -> dict:
+        if isinstance(deep_get(self.noise, match_field), list):
```

Comment:
It would be more efficient to make the dict first and then just add the 1 ip field.